x-pack/plugins/cases/public/components/actions/tags/translations.ts

```diff
@@ -51,3 +51,11 @@ export const SELECTED_TAGS = (selectedTags: number) =>
     defaultMessage: 'Selected: {selectedTags}',
     values: { selectedTags },
   });
+
+export const NO_TAGS_AVAILABLE = i18n.translate('xpack.cases.actions.tags.noTagsAvailable', {
+  defaultMessage: 'No tags available. Type to add one.',
+});
+
+export const NO_SEARCH_MATCH = i18n.translate('xpack.cases.actions.tags.noTagsMatch', {
+  defaultMessage: 'No tags available. To add a tag, type it in the query bar.',
```

Comment:
I think unless I've missed the discussion thread, this string should remain as it was in the initial commit (i.e. for the case where a search term doesn't match any tags):
```suggestion
  defaultMessage: 'No tags match your search.',
```